.github/workflows/kube-proxy.yml

```diff
@@ -0,0 +1,17 @@
+name: kube-proxy-images
+
+on:
+  schedule:
+  - cron: "0 0 * * *"
+
+jobs:
+  build:
+    runs-on: windows-2019
+    steps:
+    - uses: actions/checkout@v2
+    - uses: azure/docker-login@v1
+      with:
+        username: ${{ secrets.DOCKER_USERNAME }}
+        password: ${{ secrets.DOCKER_SECRET }}
+    - name: Build
+      run: go run kubeadm/hack/publish-kubeproxy.go kubeadm/kube-proxy
```

Comment:
i don't think `windows-2019` will include golang.
i have used [this](https://github.com/actions/setup-go) for Linux VMs in workflows, but if it doesn't work the alternative is  manually download golang for windows from their website.

Reply:
It does: https://github.com/actions/virtual-environments/blob/master/images/win/Windows2019-Readme.md#go-x64

Reply:
great!